src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -113,19 +113,28 @@ private function getContainerDeprecationLogs()
             return array();
         }
 
-        $stubs = array();
         $bootTime = filemtime($file);
+        $traceReflector = new \ReflectionProperty(\Exception::class, 'trace');
+        $traceReflector->setAccessible(true);
         $logs = array();
         foreach (unserialize(file_get_contents($file)) as $log) {
-            $log['context'] = array('exception' => new SilencedErrorContext($log['type'], $log['file'], $log['line']));
```

Comment:
why not adding trance support for the SilencedErrorContext instead of using reflection on ErrorException ?